rootfs/conf/kops/helmfile.yaml

```diff
@@ -68,6 +68,10 @@ releases:
     - name: "env.open.DISABLE_API"
       value: "true"
 
+    # show debug messages
+    - name: "env.open.DEBUG"
+      value: '{{ env "CHARTMUSEUM_DEBUG" | default "true" }}'
```

Comment:
Should this be defaulting to true here?

Reply:
Yea, I suppose we should make it false. Thanks!